README.md

```diff
@@ -224,6 +224,24 @@ Sends battery information to the server.
 | 6        | Desconnection                | 0                   |
 | 2        | Sleeping value (unused)      | 0$                  |
 
+#### BT
```

Comment:
Those are actually the names of the modules used by the devices, so that's not meant to be changed. I'll address the concern just bellow and this one by changing the reference to "Generic Button" to "Black Button (BT)" so it will be clear once the developer reads the description of the module.